### PR TITLE
fix(sonic): docker log size

### PR DESCRIPTION
**Pull request by vknabel**
for #285 (cc @iljarotar )

### CODE REVIEW

File: partition/roles/sonic/tasks/main.yaml [hunk 1]
```diff
@@ -42,6 +42,11 @@
     src: resolv.conf.j2
     dest: /etc/resolv.conf
 
+- name: render docker daemon.json
```
Comment (majst01) [inline partition/roles/sonic/tasks/main.yaml:45]: But then a handler must be triggered to restart the dockerd

Comment (vknabel) [reply partition/roles/sonic/tasks/main.yaml:45]: I am unsure if the changes in [the latest commit](https://github.com/metal-stack/metal-roles/pull/290/commits/4d10c3e055e811f2bf23c6a4c3c68a0c2b4dd6f6) are correct. Can you have another look?

**Review by robertvolkmann — CHANGES_REQUESTED** [review]
Please don't change the `daemon.json`. Every container originated from SONiC already sets:
```
"LogConfig": {
  "Type": "json-file",
  "Config": {
    "max-file": "5",
    "max-size": "2M"
  }
}
```
Please use the same settings for our containers and everything is fine, e.g.
```
--log-opt max-size=2M --log-opt max-file=5
```
The mgmt servers also miss any limits.

**Comment by robertvolkmann** [comment]
Could you also remove this nasty default for metal-core?
https://github.com/metal-stack/metal-roles/blob/master/partition/roles/metal-core/defaults/main/main.yaml#L2

**Comment by Gerrit91** [comment]
Suggestion from @robertvolkmann was also to extend the [systemd-docker-service](https://github.com/metal-stack/ansible-common/tree/master/roles/systemd-docker-service) role to include two new template variables for configuring `--log-opt max-size` and `--log-opt max-file` along with the existing `--log-driver` flag (maybe even set reasonable default in case log driver is json-file in order to prevent disks running full on the mgmt-servers for example).

The advantage would also be that we do not need to restart the Dockerd during the rollout.

What do you think?

**Comment by vknabel** [comment]
Applied your suggestions in https://github.com/metal-stack/ansible-common/pull/30
If `systemd_docker_log_driver` is not set, `json-file` will be the default. And in this case we limit the log files and size. So nothing to do here anymore.

**Comment by vknabel** [comment]
not required anymore

**Pull request closed without merging**